services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -146,32 +143,33 @@ export const ChoiceListField = ({
       form.setValue(name, selectedOptions, { shouldValidate: true });
     }
   };
-
+  function timeout(ms: number) {
+    return new Promise((resolve) => setTimeout(resolve, ms));
+  }
```

Comment:
Moved out because the prior timeout is an actual lie. Can make this a 1 line thing before merge.